cli/ballerina-cli/src/main/resources/new_cmd_defaults/manifest-app.toml

```diff
@@ -1,2 +1,12 @@
 [build-options]
 observabilityIncluded = true
+#offline = true
```

Comment:
We need to add comments explaining the sections. instead of actual values we can describe what the value should be